bond_test.go

```diff
@@ -260,6 +286,10 @@ func TestTransaction(t *testing.T) {
 	err = tx.Save(&User{Username: "Cool"})
 	assert.NoError(t, err)
 
+	// And a value that is going to be rolled back.
+	err = tx.Save(&Account{Name: "Rolled back"})
```

Comment:
This is not really going to be discarded, because the `AfterCreate` method of `Account` does not accept a context.